go.mod

```
@@ -4,9 +4,11 @@ module github.com/tetratelabs/wazero
 go 1.16
 
 require (
-	// only used in wat benchmarks
+	// only used in text benchmarks
```

I guess it's also used in bench/bench_fac_iter_test.go

```suggestion
	// only used in benchmarks
```